Codeinwp/neve
# Notices from my account page

### Summary
It seems like the notices under the MyAccount page are not wrapped in the `.woocommerce-NoticeGroup` class and this class was the one who was responsible for aligning the errors.
I added the `.woocommerce-notices-wrapper` class right next to the `.woocommerce-NoticeGroup` so the errors would align well on the account page too

### Will affect visual aspect of the product

YES

### Screenshots 
![Checkout-page-check-Check-MyAccount-errors-1-chromium-linux](https://github.com/Codeinwp/neve/assets/9929553/e1c52d94-0a8e-46f1-bd5a-6c68c010a5df)

### Test instructions
- Check that the bug is fixed on the MyAccount page
- Check that the error on the Checkout page is still looking well ( go to Checkout and try to submit without filling the form )
- Check the errors on other pages ( eg: https://neve.test/my-account/ when user is logged out and you don't fill the form ) 

## Check before Pull Request is ready:



Closes #4027
